include/spirv-tools/instrument.hpp

```diff
@@ -160,6 +159,10 @@ static const int kInst2BindlessUninitOutCnt = kInst2StageOutCnt + 3;
 
 // A buffer address unalloc error will output the 64-bit pointer in
 // two 32-bit pieces, lower bits first.
+static const int kInstBuffAddrUnallocOutDescPtrLo = kInstStageOutCnt + 1;
+static const int kInstBuffAddrUnallocOutDescPtrHi = kInstStageOutCnt + 2;
+static const int kInstBuffAddrUnallocOutCnt = kInstStageOutCnt + 3;
+
 static const int kInst2BuffAddrUnallocOutDescPtrLo = kInst2StageOutCnt + 1;
 static const int kInst2BuffAddrUnallocOutDescPtrHi = kInst2StageOutCnt + 2;
 static const int kInst2BuffAddrUnallocOutCnt = kInst2StageOutCnt + 3;
```

Comment:
If the kInst2* values are suppose to equal the corresponding kInst* value, it might be better to make them equal:

```
kInst2BuffAddrUnallocOutDescPtrLo = kInstBuffAddrUnallocOutDescPtrLo;
```

Reply:
All the kInst2* values are deprecated and will be removed as soon as the layers move away from them, which will happen as soon as this gets checked in, which will be a matter of days. The current way they are expressed represents their "true" nature, so I am not sure I want to change that. Finally, I was trying not to disrupt things too much at this point. So if we can leave them as they are for the moment, I think that would be best.

Reply:
Okay.